src/model/database.cpp

```diff
@@ -301,9 +314,23 @@ PObjectBase ObjectDatabase::CreateObject( std::string classname, PObjectBase par
 			bool create = true;
 
 			// we check the number of instances
-			if (max > 0 && CountChildrenWithSameType(parent, objType) >= max)
-				create = false;
+			int count;
+			if (objType->GetName() == wxT("sizer") || classname == "wxGridBagSizer")
+			{
+				std::set<wxString> types;
+				types.insert(wxT("sizer"));
+				types.insert(wxT("gbsizer"));
+				count = CountChildrenWithTypes(parent, types);
```

Comment:
wxFB currently requires at least C++11 (actually even C++17 is already used in one place) so you can use these language features to simplify this into `count = CountChildrenWithSameType(parent, { GetObjectType(wxT("sizer")), GetObjectType(wxT("gbsizer")) });`

Reply:
It's really great since wxWidgets itself still requires C++98.